### ISSUE TITLE
Plumed file changes shouldnt be hardcoded

### ISSUE THREAD

**Issue by ehhartmann** [hits-mbm-dev/kimmdy#442]
At the moment we automatically remove plumed variables from the plumed file if a bond is broken. This is the behavior we want if we only do homolysis but for homolysis/recombination simulations or something else, we might not want to remove the plumed variables.

https://github.com/hits-mbm-dev/kimmdy/blob/cf625b2239838f40ec3dcc6b86240f57a8258957/src/kimmdy/runmanager.py#L764-L768

Not calculating homolysis rates for broken bonds can also be solved in the homolysis plugin and is a config option at the moment.
```
            if self.config.check_bound:
                if not atomnrs[1] in top.atoms[atomnrs[0]].bound_to_nrs:
                    continue
```

What do you think about removing the `break_bond_plumed` call?

**Comment by jmbuhr** [comment]
Good point. We just have to check if removing this can lead to us trying to break already broken bonds.

**Comment by KRiedmiller** [comment]
What would be the scenario where we want a broken bond in the plumed file? If we'd remove the line, performing homolysis multiple times would break, as the same, no longer interacting bond would be extremely stretched.  
The part missing atm is adding new bonds created by recombination into the plumed file.
